docs/handbook/engineering/onboarding/how-we-work.mdx

```diff
@@ -22,8 +22,9 @@ When it comes to code review, we have a few guidelines to ensure efficiency:
 - Create a pull request in draft state as soon as possible.
 - Be proactive and review other people’s pull requests. Don’t wait for someone to ask for your review; it’s your responsibility.
 - Assign only one reviewer to your pull request.
-- **It is the responsibility of the PR owner to draft the test scenarios within the PR description. Upon review, the reviewer may assume that these scenarios have been tested and provide additional suggestions for scenarios.**
-- **Large, incomplete features should be broken down into smaller tasks and continuously merged into the main branch.**
+- Create a ticket in the current sprints project for the PR and assign it to the requested reviewer
```

Comment:
Instead of creating ticket, you can add the pr to the sprint without ticket that works too